Src/Plugins/Security/SEC101_047.MySqlConnectionStringValidator.cs

```diff
@@ -21,6 +22,17 @@ public class MySqlConnectionStringValidator : ValidatorBase
         private const string PasswordRegex = "(?i)(Pwd\\s*=\\s*(?<password>[^;]{8,128}))";
         private const string DatabaseRegex = @"(?i)(Database\s*=\s*(?<database>[^\<>:""\/\\|?;.]{1,64}))";
         private const string PortRegex = "(?i)(Port\\s*=\\s*(?<port>[0-9]{4,5}))";
+        private const string HostKey = "HOSTKEY";
+        private const string AccountKey = "ACCOUNTKEY";
+        private const string PasswordKey = "PASSWORDKEY";
+        private const string DatabaseKey = "DATABASEKEY";
+        private const string PortKey = "PORTKEY";
```

Comment:
remove this

Reply:
Removed.

---
In reply to: [575576515](https://github.com/microsoft/sarif-pattern-matcher/pull/189#discussion_r575576515) [](ancestors = 575576515)